docs/application/vscode-ext/web.md

```diff
@@ -79,142 +51,151 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
   >
   > Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
 
-  1. Open **Command Palette** and enter **Run Emulator-Manager**.
-  2. Select the profiles and versions that you want to install and click **Ok**.
+  1. In VS Code, click **View** > **Command Palette**.
+  2. In the input field of the **Command Palette** that appears, type **Run Emulator-Manager** and press **Enter**.
+  3. Select the profiles and versions for your project, and click **install**.
 
      ![Emulator Manager](media/howtoinstall-emulatormanager.png)
 
 ## Develop Applications
 
-The following section explains how to use VS Code Extension for Tizen Web to develop the applications:
+This section explains how to use VS Code Extension for Tizen Web to develop applications.
 
 ### Create Tizen Web Project
 
-To create a Tizen Web project, follow these steps:
+- To create a Tizen Web project, follow these steps:
```

Comment:
Please remove the bullet point.

To create a Tizen Web project, follow these steps:

Reply:
done.